src/stdatamodels/jwst/datamodels/reference.py

```diff
@@ -41,6 +41,23 @@ def validate(self):
             self.print_err(f'Model.meta is missing values for {to_fix}')
         super().validate()
 
+    def save(self, path, dir_path=None, *args, **kwargs):
+        """
+        Save data model.  If the 'dq' and 'dq_def' they need special
```

Comment:
minor nit, but I think there's a typo in this line. Need a verb or something after "If the dq and dq_def ...", i.e. "exist"?